docs/custom-resource-sample/report_overview1.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: "external-pipeline.redhat.com/v1"
+kind: ReportOverview
+metadata:
+  name: my-new-report-overview-object
```

Comment:
nit-pick: I'm not sure about this name maybe better `report-overview-object`

Reply:
I'll update it, but it is just an example :-D